README.md

```diff
@@ -63,6 +64,13 @@ jobs:
     params:
       path: vault-concourse-resource/secret
       prefix: secret2
+      secret_maps:
```

Comment:
secret_maps should not be required - if the secret_maps is not present, then it should copy _all_ secrets that are being tracked by the resource (found in the filetree passed in).